cli/spec/kontena/cli/main_command_spec.rb

```diff
@@ -1,45 +0,0 @@
-require "kontena/main_command"
-
-describe Kontena::MainCommand do
-
-  let(:subject) { described_class.new(File.basename($0)) }
-  describe '#subcommand_missing' do
```

Comment:
There were two different spec files for `Kontena::MainCommand`:

* `cli/spec/kontena/cli/main_command_spec.rb`
* `cli/spec/kontena/main_command_spec.rb`

Also moved the remaining specs in this file to `cli/spec/kontena/main_command_spec.rb`